src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -109,12 +118,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     if (conversationStore.isTitleTaken(conversationTitle)) {
       // conversation title is already taken, just go into that conversation instead of creating a
       // new one
+      Conversation convo = conversationStore.getConversationWithTitle(conversationTitle);
+      user.addConversation(convo);
```

Comment:
I'm probably missing something, but if the conversation exists and another user creates one with the same name, why are we adding that conversation to the user? What if they aren't supposed to be part of that conversation?